src/renderer/App.tsx

```diff
@@ -242,26 +267,54 @@ function AnalyticsBanner() {
   );
 }
 
-function GlobalContainer() {
-  // Note: NavLink is not compatible with react-router-dom's NavLink, so just add the styling
+export const GlobalContainer: FC = () => {
+  // function GlobalContainer() {
+  // The network can be set to 'devnet', 'testnet', or 'mainnet-beta'.
+  // const network = WalletAdapterNetwork.Devnet;
+
+  const validator = useAppSelector(selectValidatorNetworkState);
+  const { net } = validator;
+
+  // You can also provide a custom RPC endpoint.
+  const endpoint = useMemo(() => netToURL(net), [net]);
+
+  // @solana/wallet-adapter-wallets includes all the adapters but supports tree shaking and lazy loading --
+  // Only the wallets you configure here will be compiled into your application, and only the dependencies
+  // of wallets that your users connect to will be loaded.
+  const wallets = useMemo(
+    () => [
+      new PhantomWalletAdapter(),
+      new SlopeWalletAdapter(),
```

Comment:
Why support anything other than Phantom and Ledger (hardware wallet) here? Phantom's massively, massively popular. I think Phantom has a Ledger integration too, although I never could get it working with mine smh

Reply:
IMO we should have everything that doesn't cost us anything to support. bit worrying wrt Ledger.